src/com/dbms/DatabaseManager.java

```diff
@@ -0,0 +1,26 @@
+package com.dbms;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class DatabaseManager {
+
+	private static Connection connection;
+
+	public static synchronized Connection getConnection() {
+		try {
+			if (connection == null || connection.isClosed()) {
+				String driver = "com.mysql.cj.jdbc.Driver";
```

Comment:
This class is "DatabaseManage", but here string is specific to mysql, so if in future database changes, then modification is required here
So, apply OCP in it